backend/accounts/models.py

```diff
@@ -0,0 +1,13 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+
+class CustomUser(AbstractUser):
+    USERNAME_FIELD = EMAIL_FIELD = "email"
```

Comment:
`EMAIL_FIELD` je defaultne rovno hodnote `email`, staci jen `USERNAME_FIELD = "email"`

A taky je to neprehledne :) (radeji co radej to jedno prirazeni)

Reply:
Nenašel jsem žádný důvod proč to mít takhle. Měním na  `USERNAME_FIELD = "email"`.